### ISSUE TITLE
unable to execute WMS Client tests

### ISSUE THREAD

**Issue by vinnakota** [opengeospatial/teamengine#301]
In the latest version of [TeamEngine](http://cite.opengeospatial.org/teamengine) WMS Client test is stuck at "Executing Tests". I tried Chrome and Edge browsers, same result in both.

Using an in-house server hosting TeamEngine and it sort of works here - lists all the tests that need to execute but does not show the test status (pass/fail)

**Comment by lgoltz** [comment]
@vinnakota Did you test on [production](http://cite.opengeospatial.org/teamengine)? I've just restarted the application container and seems to work now. 

Can you explain the local failure a bit more in detail? Maybe it is the same as described in https://github.com/opengeospatial/ets-wms-client13/issues/24?

**Comment by vinnakota** [comment]
@lgoltz yes, I tested this in production. I still run in to this issue, never gets past "Executing Tests" state:

![wms_client_test](https://user-images.githubusercontent.com/4108977/34785750-eeac9a54-f5e6-11e7-87a5-ace91fe935fd.png)

* On a in-house server hosting the same version as the production the test result never updates:
![wms_client_test_enterprise](https://user-images.githubusercontent.com/4108977/34786007-9fdb2034-f5e7-11e7-910f-7e53ac1d816b.png)

* After I stop the test execution, I get this result with no other information:
![wms_client_test_enterprise1](https://user-images.githubusercontent.com/4108977/34786076-de88ce12-f5e7-11e7-88c2-111294599e51.png)

**Comment by lgoltz** [comment]
@vinnakota I've created a new issue in the WMS Client github tracker: https://github.com/opengeospatial/ets-wms-client13/issues/29

I'will close this issue as the problem with production is solved. The problem with the ETS should be  handled in the new issue.

**Comment by lgoltz** [comment]
The problem  occurred again. No errors in the log file. Problem was solved with a tomcat restart.
